3-0-spring-mvc/3-0-1-hello-spring-mvc/src/main/java/com/bobocode/mvc/controller/NoteController.java

```diff
@@ -0,0 +1,33 @@
+package com.bobocode.mvc.controller;
+
+import com.bobocode.mvc.model.Note;
+import com.bobocode.mvc.storage.Notes;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequiredArgsConstructor
+@RequestMapping("/notes")
+public class NoteController {
+
+    private final Notes notes;
+
+    @GetMapping
+    public String getNotes(Model model) {
+        model.addAttribute("notes", notes.getAll());
+        model.addAttribute("newNote", new Note());
```

Comment:
@erneman1 let's try to avoid this. I want it to be simple, and this is a bit confusing especially for beginners.
So instead of doing this, please use only `Model model` parameter in both methods `getNotes()` and `addNote()`.
And in this method please remove this line  `model.addAttribute("newNote", new Note());`
It will be simpler and much easier to understand. Thanks! :)